docker-compose.yml

```diff
@@ -16,8 +16,10 @@ x-api-common: &api-common
 services:
   api:
     <<: *api-common
-    ports:
-      - "${API_EXPOSE:-127.0.0.1:8000}:8000"
+    volumes:
+      - ./static:/opt/nutripatrol/static
```

Comment:
Have you tried it locally?
There is no "static" folder in nutripatrol so it will probably fail.
You need to add a "static" folder with an empty `.gitkeep` file inside and add this file to the repository for the folder to appear.